pkg/state/appender.go

```diff
@@ -717,6 +717,10 @@ func (a *txAppender) countExchangeScriptsRuns(scriptsRuns uint64) (uint64, error
 	return scriptsRuns, nil
 }
 
+func checkAttachment(o proto.Order) bool {
+	return o.GetAttachment().Size() != 0
+}
```

Comment:
Are you sure that this function is necessary? If yes, please, rename to `attachmentNotEmtpy` or similar.

Reply:
Not found, looks like fixed.